api/router.py

```diff
@@ -21,7 +21,7 @@ class GetTasksResponse(BaseModel):
     tasks: List[TaskModel]
 
 
-@router.get("/tasks")
+@router.get("/")
```

Comment:
including the slash here adds an extra slash at the end of the url, guessing we don't want that?
```suggestion
@router.get("")
```